new/LIT6939GadlaSosenna.xml

```diff
@@ -0,0 +1,65 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6939GadlaSosenna" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Gadla Sosǝnnā</title>
+                <title xml:lang="en" corresp="#t1">Life of Susanna</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>The text belonging to the Beta ʾƎsrāʾel (Ethiopian Jewish) literature. It stands in close relation to the Christian Homily on the 
```

Comment:
note that it is a renarration of the biblical story in `LIT3194Susanna `, simply with Daniel replaced by Michael
(see the text edited in Wurmbrand https://www.jstor.org/stable/42637851)

Reply:
One can even copy the whole story, here or as translation in the body

Have you not heard (of it) and have you not seen Susanna? Her father was a king, her mother a queen and her husband a king. After the death of her husband, kings brought to her father marriage-money for her, gold and silver, and gave same to her father, saying to him: "Give us your daughter ". Her father, the king, said to his daughter: "Come, get married ". But she refused and said to her father: " I shall not marry after the death of my husband. I have devoted myself to the Lord, my Creator ". Then came slanderous elders and spoke to Susanna: " Come, get married to (one of) us ". Bu she signed her face three times with the sign of the Lord and said to them: "You are impure and go astray from the Law of the Lord". Then the elders went to her father and said to him: " We saw your daughter fornicating and sinning with everybody ". Her father became furious about it and sent his servants to throw her into a deep [lit. " big "] pit. When the servants reached her, she prayed to the Lord and the angel Michael descended near her in the semblance of a man. He spoke to the servants: " Release her and do not throw the woman [into the pit ". His words] penetrated into their hearts and souls and they said to him: " Be it so. Maybe you can save her". Then they went to her father. He saw them, while seventy-seven kings were assembled (round him), and the look on the king's face changed. Michael said to the king: "You have not pondered your deed and have not borne in mind the Lord your God, when you delivered your handmaid (to be put to death) on account of a false accusation [lit. " word "] without a fair trial. This is not the way to judge of the Lord our God. Now do justice to this daughter of yours and let the elders be brought and stand each by himself ". Then Michael arose and said to them: " You elders, where did you see Susanna sinning with her body? ". One of them replied: "I saw her under a fig-tree ". One of them said: "I saw her in the palace ". One of them said: " I saw her in the women's apartment ". And Michael said to them: "You old witnesses who saw Susanna sinning with her body - may the Lord tear you to pieces - as there are seven divisions (in the day), at which time (was it)? ". But they could not name (the time) to Michael, because their words were lies [lit. " because of their lying words "] and their mind was set on (her) destruction. When her father saw this, (he arose) and with him arose the seventy-seven kings. They prostrated themselves before Susanna and made her a queen above them. She thus earned a double reward for her faithfulness: queenship on earth and bliss is heaven.

Reply:
Yes I added the information on LIT3194Susanna to the abstract and to the relation and quoted the translation in a div.